docs/reference/indices/split-index.asciidoc

```diff
@@ -47,6 +49,28 @@ Splitting works as follows:
 * Finally, it recovers the target index as though it were a closed index which
   had just been re-opened.
 
+[float]
+=== Why doesn't Elasticsearch support incremental resharding?
+
+Going from `N` shards to `N+1` shards, aka. incremental resharding, is indeed a
+feature that is supported by many key-value stores, which leverage the fact that
+consistent hashing only requires `1/N`-th of the keys to be relocated. However
```

Comment:
This is already great, but I feel it will be even clear if we explain that we can't just open a new empty shard and starting indexing into it - that will just create a hotspot and will result in a complicated id routing logic. This means that we have rebalance existing data. That is difficult as is already explained.